src/blocks/results/__news/results__news.css

```diff
@@ -0,0 +1,22 @@
+.results__news {
+  display: grid;
```

Comment:
I would recommend using choosing one modern technology, Flex or Grid, to use, for consistency.